src/libs/ReportUtils.js

```diff
@@ -202,7 +202,7 @@ function isMoneyRequestReport(reportOrID) {
 function sortReportsByLastRead(reports) {
     return _.chain(reports)
         .toArray()
-        .filter((report) => report && report.reportID && !isIOUReport(report))
```

Comment:
Why do we need to remove `!isIOUReport(report)` check @bernhardoj ? I remember you mentioned that lastReadTime for IOU is present after opening the report. 
I think its best to leave it as it is and add another check for `report.lastReadTime`

Reply:
Got it. Thanks for the information

Reply:
Yeah, that's correct. We used to treat them differently